src/popup.cpp

```diff
@@ -20,6 +20,9 @@ NAMESPACE_BEGIN(nanogui)
 Popup::Popup(Widget *parent, Window *parentWindow)
     : Window(parent, ""), mParentWindow(parentWindow),
       mAnchorPos(Vector2i::Zero()), mAnchorHeight(30), mSide(Side::Right) {
+    // Make sure the theme propagates to future children
+    if (mParentWindow)
+        this->setTheme(mParentWindow->theme());
```

Comment:
I got really turned around with what to do here.  Take `PopupButton`, for example.  Neither `parent` nor `parentWindow` will ever be the actual `PopupButton` instance.  Therefore when `Widget::addChild` is called, the `Theme` setting gets messed up.

It seems pretty reasonable to assume that custom theme users will set them per window, and they can always call `setTheme` manually.  I thought about adding documentation, but didn't know what to say, and feel like there is a better solution here.

It technically only works in my example repo because I'm explicitly setting the `Theme` of a window.